services/workflows-service/src/business/business.controller.external.ts

```
@@ -127,6 +132,75 @@ export class BusinessControllerExternal {
+
+        return this.businessService.updateById(
+          businessId,
+          {
+            data: {
+              ...restOfData,
+              documents: documents ? JSON.stringify(documents) : undefined,
+              additionalInfo: additionalInfo ? JSON.stringify(additionalInfo) : undefined,
+              bankInformation: bankInformation ? JSON.stringify(bankInformation) : undefined,
+              address: address ? JSON.stringify(address) : undefined,
+              shareholderStructure:
+                shareholderStructure && shareholderStructure.length
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)